test/onRequest.js

```diff
@@ -1,342 +1,144 @@
 'use strict';
 process.env.NODE_ENV = 'test';
-var expect = require('expect.js');
-var rewire = require('rewire');
-var plugin = rewire('../src/plugin.js');
+const expect = require('expect.js');
+const rewire = require('rewire');
+const plugin = rewire('../src/plugin.js');
 
+const request = {
+  info: {
+    received: new Date()
+  },
+  method: 'get',
+  response: {
+    statusCode: 200
+  },
+  query: {
+    firstName: 'Doron',
+    last_Name: 'Doron',
```

Comment:
Wasn't it Segal?

Reply:
it was... but this was he wrote in the original tests. 

Like I said, fix dumb tests